<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Прокалка электродов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Графики</string>
    </property>
    <addaction name="actionGraph"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Печи</string>
    </property>
    <addaction name="actionOven"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Отчеты</string>
    </property>
    <addaction name="actionDryKvo"/>
    <addaction name="actionDryEnerg"/>
    <addaction name="actionDryMas"/>
    <addaction name="separator"/>
    <addaction name="actionDryEnergKg"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionGraph">
   <property name="text">
    <string>Графики температуры</string>
   </property>
  </action>
  <action name="actionOven">
   <property name="text">
    <string>Конфигурация печей</string>
   </property>
  </action>
  <action name="actionDryKvo">
   <property name="text">
    <string>Количество прокалок</string>
   </property>
  </action>
  <action name="actionDryEnerg">
   <property name="text">
    <string>Расход энергии</string>
   </property>
  </action>
  <action name="actionDryMas">
   <property name="text">
    <string>Прокаленные электроды</string>
   </property>
  </action>
  <action name="actionDryEnergKg">
   <property name="text">
    <string>Расход энергии на кг электродов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
